kinto/tests/support.py

```diff
@@ -42,6 +42,9 @@ def _get_test_app(self, settings=None):
 
     def get_app_settings(self, additional_settings=None):
         settings = cliquet_support.DEFAULT_SETTINGS.copy()
+        settings['cliquet.cache_backend'] = 'cliquet.cache.memory'
+        settings['cliquet.storage_backend'] = 'cliquet.storage.memory'
+        settings['cliquet.permission_backend'] = 'cliquet.permission.redis'
```

Comment:
This was to be able to use ``redis-cli monitor`` to debug the permission backend.
I supose that we should put them all on the memory backend now that tests are passing.